src/EnergyPlus/DualDuct.cc

```diff
@@ -886,64 +785,42 @@ namespace DualDuct {
         // METHODOLOGY EMPLOYED:
         // Uses the status flags to trigger events.
 
-        // Using/Aliasing
-        using DataZoneEquipment::CheckZoneEquipmentList;
-        using Psychrometrics::PsyRhoAirFnPbTdbW;
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         int HotInNode;
         int ColdInNode;
         int OAInNode; // Outdoor Air Inlet Node for VAV:OutdoorAir units
         int RAInNode; // Reciruclated Air Inlet Node for VAV:OutdoorAir units
         int OutNode;
-        // static Array1D_bool MyEnvrnFlag;
-        // static Array1D_bool MySizeFlag;
-        // static Array1D_bool MyAirLoopFlag;
         int Loop;          // Loop checking control variable
         Real64 PeopleFlow; // local sum variable, m3/s
 
-        // Do the Begin Simulation initializations
-        if (state.dataDualDuct->InitDualDuctMyOneTimeFlag) {
-
-            // MyEnvrnFlag.allocate(NumDDAirTerminal);
-            // MySizeFlag.allocate(NumDDAirTerminal);
-            // MyAirLoopFlag.dimension(NumDDAirTerminal, true);
-            // MyEnvrnFlag = true;
-            // MySizeFlag = true;
-            state.dataDualDuct->MassFlowSetToler = DataConvergParams::HVACFlowRateToler * 0.00001;
-
-            state.dataDualDuct->InitDualDuctMyOneTimeFlag = false;
-        }
-
         if (!state.dataDualDuct->ZoneEquipmentListChecked && state.dataZoneEquip->ZoneEquipInputsFilled) {
             state.dataDualDuct->ZoneEquipmentListChecked = true;
             // Check to see if there is a Air Distribution Unit on the Zone Equipment List
             for (Loop = 1; Loop <= state.dataDualDuct->NumDDAirTerminal; ++Loop) {
                 if (this->ADUNum == 0) continue;
-                if (CheckZoneEquipmentList(state, "ZONEHVAC:AIRDISTRIBUTIONUNIT", state.dataDefineEquipment->AirDistUnit(this->ADUNum).Name))
+                if (DataZoneEquipment::CheckZoneEquipmentList(
+                        state, "ZONEHVAC:AIRDISTRIBUTIONUNIT", state.dataDefineEquipment->AirDistUnit(this->ADUNum).Name))
                     continue;
                 ShowSevereError(state,
                                 "InitDualDuct: ADU=[Air Distribution Unit," + state.dataDefineEquipment->AirDistUnit(this->ADUNum).Name +
                                     "] is not on any ZoneHVAC:EquipmentList.");
                 if (this->DamperType == DualDuctDamper::ConstantVolume) {
-                    ShowContinueError(
-                        state, "...Dual Duct Damper=[" + state.dataDualDuct->cCMO_DDConstantVolume + ',' + this->Name + "] will not be simulated.");
+                    ShowContinueError(state,
+                                      "...Dual Duct Damper=[" + std::string(cCMO_DDConstantVolume) + ',' + this->Name + "] will not be simulated.");
```

Comment:
Gah, need to clean out a few of these that I missed when following up with my format changes.